//
// Copyright 2022 Ant Group Co., Ltd.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//   http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
//

syntax = "proto3";

package spu.psi.proto;

message SizeProto {
  uint64 input_size = 1;
}

message PsiDataBatchProto {
  uint32 item_num = 1;
  bytes flatten_bytes = 2;
  bool is_last_batch = 3;
}

message StrItemsProto {
  repeated string items = 1;
}